testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -1788,6 +1788,63 @@ QUnit.module('selection', moduleSetup, () => {
     });
 });
 
+QUnit.module('selectByClick option', ()=> {
+    QUnit.test('selectByClick option should be changed during initialization', function(assert) {
+        const list = $('#list').dxList({ items: [1, 2], selectByClick: false }).dxList('instance');
+        const isSelectByClick = list.option('selectByClick');
+
+        assert.strictEqual(isSelectByClick, false);
+    });
+
+    QUnit.test('selectByClick option should be changed after initialization', function(assert) {
+        const list = $('#list').dxList({ items: [1, 2] }).dxList('instance');
+
+        list.option('selectByClick', false);
+
+        const isSelectByClick = list.option('selectByClick');
+
+        assert.strictEqual(isSelectByClick, false);
+    });
+
+    QUnit.test('onSelectionChanged event shouldn`t call on item click if selectByClick is false', function(assert) {
+        const actionCount = sinon.spy();
```

Comment:
1. This variable value is not a count, it's a spy.
2. Name is too common to me.
```suggestion
        const onSelectionChangedSpy = sinon.spy();
```

Reply:
Fixed